.spectral.yml

```diff
@@ -0,0 +1,8 @@
+extends: [[spectral:oas, all]]
+rules:
```

Comment:
Very impressive tool.
After looking into the [OpenAPI rules](https://meta.stoplight.io/docs/spectral/docs/reference/openapi-rules.md) I think we should add also:
* operation-operationId-unique
* operation-2xx-response
* path-params
* no-$ref-siblings
* openapi-tags
* path-keys-no-trailing-slash

Reply:
Agree, I wanted to start this commit with a minimum and add more later